Classes/Core/Functional/FunctionalTestCase.php

```diff
@@ -496,15 +497,20 @@ private function createServerRequest(string $url, string $method = 'GET'): Serve
     {
         $requestUrlParts = parse_url($url);
         $docRoot = $this->instancePath;
+
+        // @todo: Remove when dropping support for v12
+        $hasConsolidatedHttpEntryPoint = class_exists(CoreHttpApplication::class);
```

Comment:
that's a simple and straight b/w compat detection strategy. fine with me. and it's smart since v13 core without the entry-point patch does not fail (which would happen when going for a typo3Version check).